readme.md

```diff
@@ -172,6 +172,13 @@ Default: `false`
 
 Emulate preference of dark color scheme.
 
+##### launchOptions
+
+Type: `object`\
+Default: `{}`
+
+Specify launch options to be passed to the underlying library and consequently to `puppeteer.launch()`.
```

Comment:
This should be documented like it's done in `capture-websites`.

Reply:
Should be more appropriate now.